examples/resourceServer/server/index.js

```diff
@@ -7,7 +7,7 @@ function initializeClients() {
         {
             clientId: 'clientApplication', // unique identifier.
             active: true, // whether this client is enabled or not.
-            redirectUri: 'http://localhost:3200/_oauth/MeteorOAuth2Server', // provided by the client.
+            redirectUri: 'http://localhost:3100/_oauth/MeteorOAuth2Server', // provided by the client.
```

Comment:
The client_id is a good catch and was a mistake. There is a pending pull request that fixes this. Refer to the previous comment about the port number.